src/web/lib/hakuneko/frontend@classic-dark/menu.html

```diff
@@ -270,7 +282,7 @@
              * otherwise the window cannot be closed anymore (electron does not show the confirmation).
              */
             openWindow(event) {
-                let popup = window.open(event.target.dataset.href, '', 'frame=true,nodeIntegration=no');
+                let popup = window.open(event.target.dataset.href, '', 'nodeIntegration=no');
```

Comment:
I think without the frame there will be no minimize, maximize and close buttons on the window and as a result the window cannot be dismissed by the user